src/extension/abstract.js

```diff
@@ -21,16 +23,18 @@ export class AbstractExtension {
     this._type = value
   }
 
+  // Content.
+
   /**
-   * The content of this extension.
+   * The Custom XML object.
```

Comment:
I don't think "custom" is the right word. Also, it shouldn't be capitalized. How about just "the Extension XML element"?